File: app/src/main/java/com/zhxh/xdialog/MainActivity.java
package com.zhxh.xdialog;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btnDialog).setOnClickListener(v -> startActivity(new Intent(MainActivity.this, DialogActivity.class)));
        findViewById(R.id.btnDialogFragment).setOnClickListener(v -> startActivity(new Intent(MainActivity.this, TestActivity.class)));
    }
}

File: app/src/main/java/com/zhxh/xdialog/MainActivity.kt
package com.zhxh.xdialog

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<View>(R.id.btnToast).setOnClickListener { v -> startActivity(Intent(this@MainActivity, ToastActivity::class.java)) }
        findViewById<View>(R.id.btnDialog).setOnClickListener { v -> startActivity(Intent(this@MainActivity, DialogActivity::class.java)) }
        findViewById<View>(R.id.btnDialogFragment).setOnClickListener { v -> startActivity(Intent(this@MainActivity, TestActivity::class.java)) }
    }
}
